src/WebJobs.Script/Description/DotNet/FunctionSignature.cs

```diff
@@ -67,7 +67,7 @@ public string MethodName
             }
         }
 
-        private static bool AreParametersEquivalent(IParameterSymbol param1, IParameterSymbol param2)
+        private static bool AreParametersEquivalent(FunctionParameter param1, FunctionParameter param2)
```

Comment:
Good catch. This was existing code that I just updated but yes, it should absolutely be moved to the new `FunctionParameter` class. I'll do that before merging in.